src/Button/Button.Component.js

```diff
@@ -151,15 +159,7 @@ const clickBtnHandler = btn => {
             <Separator />
 
             <h2>Button Types</h2>
-            <Description>
-                <strong> - Action Button</strong>: The default button <br />
-                <strong> - Standard Button</strong>: Neutral or informative color <br />
-                <strong> - Positive Button</strong>: Used for positive actions such as approved, ok, yes. <br />
-                <strong> - Medium Button</strong>: Used for warnings or alert <br />
-                <strong> - Negative Button</strong>
-                :Used for negative actions such as decline, cancel, no.
-                <br />
-            </Description>
+            <Description children={buttonTypesDescription} />
```

Comment:
Although I know this works, is it clearer/cleaner/more standard to just place the variable inside the JSX tags?
```jsx
<Description>{buttonTypesDescription}</Description>
```
Same comment on a few other instances below.